src/ValueHandler/ProductOptionValueHandler.php

```diff
@@ -41,6 +47,16 @@ public function __construct(
             );
         }
         $this->translationLocaleProvider = $translationLocaleProvider;
+        if ($translator === null) {
+            trigger_deprecation(
+                'webgriffe/sylius-akeneo-plugin',
+                '1.15',
+                'Not passing a translator to %s is deprecated and will not be possible anymore in %s',
+                __CLASS__,
+                '2.0'
+            );
+        }
```

Comment:
```suggestion
```

BC break is allowed on master.

Reply:
Yes, but I will keep this kind of changes on the #130.

Reply:
Ok!